docs/table/index.rst

```diff
@@ -364,6 +364,14 @@ Implementation
 
    implementation_details.rst
 
+Tips and Tricks
+---------------
+   
+.. toctree::
+   :maxdepth: 2
+   
+   tips_and_tricks.rst
```

Comment:
I'm not sure a separate tips and tricks page is the best place for the new documentation. Wouldn't https://docs.astropy.org/en/latest/table/mixin_columns.html#quantity-and-qtable or https://docs.astropy.org/en/latest/table/io.html be more appropriate?